src/compiler/index.js

```diff
@@ -14,25 +14,31 @@
  * limitations under the License.
  */
 
+import * as compiler from '@ampproject/bento-compiler';
+
+import {getBuilders} from './builders';
+
 /**
- * Returns the HTML for an AMP Document with eligible components server-rendered.
- *
- * @param {string} html
- * @return {string}
+ * @typedef {*} TreeProto
+ * TODO: replace with definition in bento-compiler.
+ */
+
+/**
+ * @typedef {{
+ *   runtimeVersion: string,
+ *   extensionList: Array<{name: string, version:string}>
+ * }} VersionData
  */
-function compileHtml(html) {
-  return html;
-}
 
 /**
  * Returns the AST for an AMP Document with eligible components server-rendered.
  *
- * @param {*} ast
- * @return {*}
+ * @param {TreeProto} ast
+ * @param {VersionData} versionData
```

Comment:
nit/google style recommendation: avoid `data` in identifiers, adds nothing. `VersionMap` or `ExtensionVersions`

Reply:
Done!